lib/ads/client_side_ad_manager.js

```diff
@@ -96,6 +99,15 @@ shaka.ads.ClientSideAdManager = class {
     this.adsLoader_.requestAds(imaRequest);
   }
 
+  /**
+   * @param {!google.ima.AdsRenderingSettings} adsRenderingSettings
+   */
+  updateAdsRenderingSettings(adsRenderingSettings) {
+    if (this.imaAdsManager_) {
```

Comment:
You should also change the value of `this.adsRenderingSettings_`, so that this method will still work if called before we receive an `ADS_MANAGER_LOADED` event and create `this.imaAdsManager_`.